lib/services/panduit-obm-service.js

```diff
@@ -143,14 +150,15 @@ function panduitObmServiceFactory(
         });
     };
 
-    PanduitObmService.prototype._runInternal = function (command, host, community, file) {
+    PanduitObmService.prototype._runInternal = function (command, host, community, file, retries) {
         return this.run({
-            command: file || 'snmpset',
```

Comment:
'snmpset' in "file" parameter was the last parameter and it can be set as a default one. Now I add "retires", so I don't view "file" as a default value, and added "snmpset" in this file when calling this _runInternal function. It works when this function is called in only one or two places. Thank you for reminding me that there are some other ways to add a new parameter with less code changes and confusion.